NitroxClient/MonoBehaviours/Multiplayer.cs

```diff
@@ -137,8 +137,9 @@ public void InitMonoBehaviours()
             gameObject.AddComponent<ThrottledBuilder>();
 
             // UI.
+            gameObject.AddComponent<KickedModal>();
             gameObject.AddComponent<LostConnectionModal>();
-            gameObject.AddComponent<PlayerKickedModal>();
+            gameObject.AddComponent<ServerStoppedModal>();
```

Comment:
Maybe we can be more flexible than adding by hand those modals ?

Reply:
What about some class that detects all the Modals from this solution and registers them automatically ?

Reply:
You can already do it with dependency injection

Reply:
Yea this is what I meant, something like this [NitroxPatchesModule](https://github.com/SubnauticaNitrox/Nitrox/blob/master/NitroxPatcher/Modules/NitroxPatchesModule.cs)